platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -1,150 +1,161 @@
 1. Attributes of successful platforms
 1. Attributes of successful platform teams
 1. Challenges when implementing platforms
 1. How to measure the success of platforms
 1. Capabilities of platforms
 
-## What is a platform
-
-A platform is a layer that provides common supporting capabilities and services
-for many applications and use cases. Such a platform provides consistent user
-experiences for getting, using and managing its capabilities and services,
-including Web portals and pages, scenario-specific code templates, automatable
-APIs and command-line tools.
-
-According to Atlassian [[1]], "platform teams create capabilities that can be
-used by numerous stream-aligned teams with little overhead.... platform teams
-minimize resources and cognitive load of the stream-aligned team... platform
-teams can create a cohesive experience that spans across different user
-experiences or products."
-
-According to Martin Fowler and Evan Bottcher [[2]], "a digital platform is a
-foundation of self-service APIs, tools, services, knowledge and support which
-are arranged as a compelling internal product. Autonomous delivery teams can
-make use of the platform to deliver product features at a higher pace, with
-reduced coordination."
+## Why have internal platforms gained popularity
 
-Platforms are particularly relevant for cloud-native computing, which typically
-separates supporting capabilities from application-specific logic more than
-previous paradigms. In such environments resources like databases and object
-stores, message queues and brokers, observability collectors and dashboards,
-user directories and authentication systems, task runners and reconcilers and
-more are managed independently and integrated into applications running in
-containers and machines. A platform for cloud-native computing provides these to
-many teams in ways that make them easy to integrate in applications and systems.
-
-At its most basic, a platform provides consistent experiences to application
-developers for acquiring and using individual services such as a database system
-or a secret store. A more advanced platform also _composes_ these capabilities
-into experiences and templates fit for key scenarios like web application
-development or data analysis. For example, some organizations today provide
-_developer_ platforms that compose resources required for application
-development such as compute environments, pipeline runners, database systems,
-messaging queues, identities and telemetry collectors. An organization may also
-provide a data analysis ("MLOps") platform that composes data ingestion and
-transformation services, model training services, model deployment pipelines and
-model servers.
-
-By offering consistent experiences for individual and/or scenario-oriented sets
-of capabilities, platforms make it easy for their users to deliver valuable
-products.
+DevOps provides enterprises a level of agility that has not previously been 
```

Comment:
Thanks for these paragraphs @abangser, they provide a nice transition into the section now that it's moved to be first. I'm inspired by how you tied "platforms" into the evolution of computing and related it to the DevOps movement, I think that's pretty accurate and fits with the first line in the introduction.

I'm thinking though that perhaps we shouldn't use the term DevOps here, perhaps instead describing the history more generally? Also thinking about how we might transition from discussing "DevOps" to positing the value of "platforms" in the next paragraph. WDYT of something like the following? Thanks!

> Process improvements over the past 2-3 decades have significantly increased the agility of software application and product teams, offering them flexible services for both infrastructure like compute, network and storage and developer services like builds, tests, delivery and observability. But this autonomy and process improvement have also gradually shifted more and more responsibility for supporting services to product teams, forcing them to spend more and more time and cognitive energy on infrastructural concerns and reducing their time to produce value relevant to their organization.
>
> This business need has motivated enterprises to implement platforms for cloud-native computing. Some benefits of such platforms include the following:

Reply:
I knew you could help refine this! Thanks so much for building off these ideas in a concise and clear way. I really like how you generalised it 👏